includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -520,6 +573,60 @@ public function should_show_save_option() {
 		return $this->is_reusable() && $this->is_saved_cards_enabled();
 	}
 
+	/**
+	 * Returns the payment method's limits per currency.
+	 *
+	 * @return int[][][]
+	 */
+	public function get_limits_per_currency(): array {
```

Comment:
Hey! Curious about this. Why do we expose the `limits_per_currency` protected property publicly when it's only used within the class?

I see that we use this public function in the unit tests to mock its return value, but just that.
https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3083/files#diff-358342362ae0a24ddc1254fd5f6632bff220c4147eaafffa3d5f0b7654ce3c11R124

Reply:
Hi! Yes, that's the reason why the method is public. Do you think this is bad?

Reply:
Gotcha, thanks for clarifying!

Nothing will explode, but it's not ideal. It'd be great if we could test these changes without having to expose the property just for the tests. But I wouldn't hold this PR just because of that.

It's probably a smell on either the tests or how we're structuring these checks. Worth checking separately.

Reply:
I tried to remove the method, but then I could not mock or override the protected property. I could add a new option (and override it), like we do for `testmode`, but I think that would be worse 😬 .